src/GraphQL/Validation/IValidationResult.cs

```diff
@@ -1,7 +1,11 @@
+using GraphQL.Execution;
+using GraphQLParser.AST;
+
 namespace GraphQL.Validation
 {
     /// <summary>
     /// Contains a list of the validation errors found after validating a document against a set of validation rules.
+    /// If the document passes validation, this will also contain the set of parsed variables and argument values.
```

Comment:
```suggestion
    /// If the document passes validation, this will also contain the set of parsed variables and argument values for fields and applied directives.
```